app/javascript/src/hyacinth_v1/components/digital_objects/search/FacetSidebar.spec.js

```diff
@@ -0,0 +1,33 @@
+import renderer from 'react-test-renderer';
+import FacetDropdown from './facet_sidebar/FacetDropdown';
+import FacetSidebar from './FacetSidebar';
+
+jest.mock('./facet_sidebar/FacetDropdown', () => ({
+  __esModule: true, default: jest.fn(() => 'FacetDropdown'),
+}));
```

Comment:
I might be wrong about this, but I _think_ this mock will leak across other tests/files.  The `jest.spyOn` method is a good alternative to `jest.mock` because it's easy to reset (and generally you'd want to call it in a beforeEach), but I don't think we can use it here because you're mocking the default export of a module rather than a function on an object.  Let's talk more about this tomorrow if you have a chance.